webapp/tests/forms/steps/lotse/test_personal_data_steps.py

```diff
@@ -573,3 +577,79 @@ def test_if_required_value_is_not_give_validation_should_be_failure(self, new_te
                 StepPersonBHasDisability.name, True, ImmutableMultiDict(data))
             form = step.render_info.form
             assert form.validate() is False
+
+class TestPauschBetragPersonAValidation:
+    def test_if_required_value_is_given_then_validation_should_be_success(self, new_test_request_context):
+        data = MultiDict({'person_a_has_disability':'yes', 'person_a_requests_pauschbetrag': 'no'})
+        with new_test_request_context(form_data=data) as req:
+            req.session = SecureCookieSession(
+                {_LOTSE_DATA_KEY: create_session_form_data(data)})
+            step = LotseStepChooser().get_correct_step(
+                StepPauschbetragPersonA.name, True, ImmutableMultiDict(data))
+            form = step.render_info.form
+            assert form.validate() is True
+
+    def test_if_required_precondition_person_is_not_satisfied_return_should_be_a_redirect_to_person_a_has_disability(self, new_test_request_context):
+        data = MultiDict({})
+        with new_test_request_context(form_data=data) as req:
+            req.session = SecureCookieSession(
+                {_LOTSE_DATA_KEY: create_session_form_data(data)})
+            step = LotseStepChooser().get_correct_step(
+                StepPauschbetragPersonA.name, True, ImmutableMultiDict(data))
+            assert step.redirection_step_name == 'person_a_has_disability'
+            
+    def test_if_required_precondition_person_a_has_disability_yes_is_not_satisfied_return_should_be_a_redirect_to_person_a_has_disability(self, new_test_request_context):
+        data = MultiDict({'person_a_has_disability':'no', 'person_a_requests_pauschbetrag': 'no'})
```

Comment:
```suggestion
        data = MultiDict({'person_a_has_disability':'no'})
```
Could we leave this empty because the pauschbetrag should not be set yet and we are performing a GET if I understand it correctly?